RUFAS/routines/manure/gas_emissions/gas_emissions.py

```diff
@@ -647,3 +648,177 @@ def calc_methane_emission_from_anaerobic_lagoon(cls, manure_volatile_solids: flo
         MS = GasEmissionConstants.FRACTION_OF_HANDLED_MANURE
         MF = GasEmissionConstants.METHANE_FACTOR
         return Bo * MCF * MS * MF * manure_volatile_solids
+
+    @classmethod
+    def calc_mcf(cls, ambient_barn_temp: float) -> float:
+        """
+        Calculate the Methane Conversion Factor (MCF) using the exponential function:
+        MCF(T) = MCF_CONSTANT_A * e^(MCF_CONSTANT_B * T)
+
+        Parameters
+        ----------
+        ambient_barn_temperature : float
```

Comment:
```suggestion
        ambient_barn_temp : float
```
Keep it consistent with the function definition?